assertj-core/src/test/java/org/assertj/core/presentation/StandardRepresentation_iterable_format_Test.java

```diff
@@ -260,6 +264,22 @@ public void should_use_overridden_toString_over_iterable_representation_in_colle
     then(formatted).isEqualTo("[{\"a\":1}]");
   }
 
+  @Test
+  void should_format_big_list() {
+    // GIVEN
+    int elementsPerArray = 1000;
+    List<int[]> numbers = new ArrayList<>();
+    for (int i = 0; i < 1 << 20; i++) {
```

Comment:
I guess `1 << 20` is too much for the current github actions tests memory settings since the test ends with an OOM.

Reply:
Oh, OK. It's pretty extreme, so you could dial it back to 2^15 elements, with each element an array of 500, or something like that.